pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -237,7 +248,7 @@ func (e *BootstrapSigner) updateConfigMap(cm *v1.ConfigMap) {
 
 // getConfigMap gets the ConfigMap we are interested in
 func (e *BootstrapSigner) getConfigMap() *v1.ConfigMap {
-	configMap, exists, err := e.configMaps.GetByKey(e.configMapKey)
+	configMap, err := e.configMapLister.ConfigMaps(e.configMapNamespace).Get(e.configMapName)
 
 	// If we can't get the configmap just return nil. The resync will eventually
```

Comment:
I think the previous code would return error=nil and exists=false if the ConfigMap didn't exist. In that case, it wouldn't call `utilruntime.HandleError` because err is nil.

The new code will return a NotFound error if it doesn't exist. Do you want to check for that and return nil without calling HandleError in that case?

Reply:
Great， updated